ONSdigital/ssdc-rm-docker-dev
# Remove case api

# Motivation and Context

We no longer use case api
# What has changed



References to the case api have been removed
# How to test?



Check out branch and do a make up. Check the case api doesn't run. Run ATs from branch on the ticket
# Links



https://jira.ons.gov.uk/browse/SDCSRM-590
# Screenshots (if appropriate):
